lib/Table/TableStyle.ts

```diff
@@ -513,9 +537,14 @@ export default class TableStyle {
         ? this.timeColumn.valuesAsJulianDates.maximum
         : sortedStartDates[sortedStartDates.length - 1];
 
-    return startDates.map(date => {
+    const finishDates: (JulianDate | null)[] = new Array(
```

Comment:
what's the benefit of doing it this way over the old map?

Reply:
Performance, we sometimes have a million dates or so